doc/guides/west/build-flash-debug.rst

```diff
@@ -22,6 +22,30 @@ directly delegate to West.
    .. contents::
       :local:
 
+.. _west-boards:
+
+Listing boards: ``west boards``
+*******************************
+
+.. tip:: Run ``west boards -h`` for additional help.
```

Comment:
I'd delete this since you've already mention ``-h`` below (no need to mention it twice)